lib/openstudio_measure_tester/runner.rb

```diff
@@ -96,6 +102,8 @@ def post_process_results(original_results_directory = nil)
       # call the create dashboard command now that we have results
       dashboard
 
+      github_actions_report if ENV["GITHUB_ACTIONS"]
```

Comment:
No-op unless you define `GITHUB_ACTIONS` env var. This is perfectly safe to define it locally for testing.